src/main/java/org/sunbird/migrate/service/impl/UserMigrationServiceImpl.java

```diff
@@ -0,0 +1,247 @@
+package org.sunbird.migrate.service.impl;
+
+import org.apache.commons.collections.MapUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.elasticsearch.common.recycler.Recycler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.sunbird.common.model.SBApiResponse;
+import org.sunbird.common.service.OutboundRequestHandlerServiceImpl;
+import org.sunbird.common.util.CbExtServerProperties;
+import org.sunbird.common.util.Constants;
+import org.sunbird.common.util.PropertiesCache;
+import org.sunbird.core.config.PropertiesConfig;
+import org.sunbird.migrate.service.UserMigrationService;
+import org.sunbird.profile.service.ProfileService;
+
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
+
+@Service
+public class UserMigrationServiceImpl implements UserMigrationService {
+
+    private Logger log = LoggerFactory.getLogger(getClass().getName());
+
+    @Autowired
+    OutboundRequestHandlerServiceImpl outboundRequestHandlerService;
+
+    @Autowired
+    CbExtServerProperties serverConfig;
+
+    @Autowired
+    PropertiesConfig propertiesConfig;
+
+    @Autowired
+    ProfileService profileService;
+
+    @Override
+    public SBApiResponse migrateUsers() {
+        SBApiResponse response = new SBApiResponse(Constants.API_USER_MIGRATION);
+        StringBuilder url = new StringBuilder(propertiesConfig.getLmsServiceHost()).append(propertiesConfig.getLmsUserSearchEndPoint());
+        log.info("printing user search Url: {}", url);
+
+      try {
+          Map<String, Object> request = userSearchRequestBody();
+          Map<String, Object> updateResponse = outboundRequestHandlerService.fetchResultUsingPatch(url.toString(), request, null);
```

Comment:
Why search request body is used with fetchResultsUsingPatch ??

Also this is wrong implementation which calls the API only once... what will happen if the number of records are 10K in a day... this API will return 250 records only... 
you need to call once -- check the count in the response.. then process the all the response in the content and proceed the search API again with offset as processed number of items... like pagination...